README.md

```diff
@@ -193,4 +193,10 @@ You may then repeat the --init-token flow to set up a fresh HSM instance.
 
 ## login
 
-Coming soon...
+You may login using a serial number from the `list` command.
```

Comment:
Is it worth mentioning that --serial is optional?  Its a bit of a weird feature because you don't have control over which is used if you don't specify, so the feature generally only makes sense when the number of tokens == 1.  But it is there, nonetheless.

Reply:
Yes, I can add that as I normally do when I only have 1